bin/build.sh

```diff
@@ -39,15 +43,13 @@ docker save --output="waffleio-takeout/waffle.io-hooks.tar" quay.io/waffleio/waf
 docker save --output="waffleio-takeout/waffle.io-migrations.tar" quay.io/waffleio/waffle.io-migrations
 docker save --output="waffleio-takeout/waffle.io-rally-integration.tar" quay.io/waffleio/waffle.io-rally-integration
 
-cp install.sh waffleio-takeout/
-cp waffleio-env.list waffleio-takeout/
+cp bin/install.sh waffleio-takeout/
```

Comment:
If we move, you would be unable to run the install more than once. Seems not great.